app/src/main/java/com/fantastic/bookxchange/activities/UserActivity.java

```diff
@@ -99,6 +105,15 @@ private void setupTab() {
     @Override
     public void onClickListener(Book book) {
         //TODO Itent to Book Detail page
+        Log.i("Book", book.getTitle().toString());
+        Toast.makeText(this, book.getTitle().toString(), Toast.LENGTH_LONG).show();
```

Comment:
You can use the toast function that is on Base activity

Reply:
Ok, I forgot that